app/views/cars/_form.html.haml

```diff
@@ -2,4 +2,5 @@
   = f.input :brand, as: :string
   = f.input :model, as: :string
   = f.input :registration_number, as: :string
-  = f.submit
+  = f.button :submit
+  = link_to t('back'), maintenances_path, class: 'btn btn-secondary'
```

Comment:
I think it's better to use the ':back' option instead of hardcoded path. 
``` = link_to t('back'), :back, class: 'btn btn-secondary' ```

For example, if you clicking a particular link in 'maintenance' section on  Dashboard, the shown page doesn't work properly after using the 'back' button.